packages/flame_texturepacker/README.md

```diff
@@ -0,0 +1,89 @@
+<!-- markdownlint-disable MD013 -->
+<p align="center">
+  <a href="https://flame-engine.org">
+    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
+  </a>
+</p>
+
+<p align="center">
+A flame plugin to import sprite sheets generated by [Gdx Texture Packer][2] and
+[Code and Web Texture Packer][1]
+</p>
+
+<p align="center">
+  <a title="Pub" href="https://pub.dev/packages/flame_texturepacker" ><img src="https://img.shields.io/pub/v/flame_texturepacker.svg?style=popout" /></a>
+  <a title="Test" href="https://github.com/flame-engine/flame/actions?query=workflow%3Acicd+branch%3Amain"><img src="https://github.com/flame-engine/flame/workflows/cicd/badge.svg?branch=main&event=push"/></a>
+  <a title="Discord" href="https://discord.gg/pxrBmy4"><img src="https://img.shields.io/discord/509714518008528896.svg"/></a>
+  <a title="Melos" href="https://github.com/invertase/melos"><img src="https://img.shields.io/badge/maintained%20with-melos-f700ff.svg"/></a>
+</p>
+
+---
+<!-- markdownlint-enable MD013 -->
+
+<!-- markdownlint-disable-next-line MD002 -->
+# flame_texturepacker
+
+
+## Install from Pub
+
+```console
+flutter pub add flame_texturepacker
+```
```

Comment:
Do we include such instructions for other bridge packages? Or is it implied at this point?

Reply:
We're not consistent unfortunately, we should create some form of template for bridge libraries.